resource/css/skosmos.css

```diff
@@ -275,6 +279,55 @@ body {
   #headerbar input {
     background-color: var(--search-bg);
     border: 1px solid var(--search-border);
+    border-radius: 0;
+  }
+
+  #headerbar span.dropdown {
+    width: 25rem;
+  }
+
+  #headerbar .autocomplete {
+    position: relative;
+  }
+
+  #headerbar .autocomplete-result {
+    border: 1px solid var(--search-border);
+    margin: auto;
+  }
+
+  #headerbar .autocomplete-result a {
+    color: var(--vocab-link);
+    text-decoration: none;
+  }
+  
+  #headerbar .autocomplete {
+    position: relative;
+  }
+  
+  #headerbar .autocomplete-results {
+    padding: 0;
+    margin: 0;
+    border: 1px solid #eeeeee;
+    height: 120px;
+    min-height: 1em;
+    max-height: 6em;    
+    overflow: auto;
+  }
+
+  #headerbar .autocomplete-result {
+    list-style: none;
+    text-align: left;
+    padding: 4px 2px;
+    cursor: pointer;
+  }
+
+  #headerbar .autocomplete-result:hover {
+    background-color: #4AAE9B;
```

Comment:
Why this green shade? It doesn't seem to fit in with the rest of the current palette. I would expect dark blue or gray... Also the contrast between the blue link text and the greenish background is quite low.

Maybe we just need a proper visual design for the autocomplete list...